src/python/WMCore/Database/ExecuteDAO.py

```diff
@@ -160,16 +160,22 @@ def __init__(self, logger=None, configFile=None,
             config.CoreDatabase.dialect = getBackendFromDbURL(os.getenv("DATABASE"))
             config.CoreDatabase.socket = os.getenv("DBSOCK", None)
```

Comment:
I think this DBSOCK line should only be executed if we find out that the dialect is mysql

Reply:
No need to add any additional check here for the database type. This is tried to be fetched from the environment, and this environment variable is simply not going to be set for agents connecting to Oracle.